popsycle/synthetic.py

```diff
@@ -775,15 +781,20 @@ def _make_comp_dict(iso_dir, log_age, currentClusterMass, star_dict, next_id,
             ##########
             # Add kicks to NSs and BHs.
             ##########
+            
             NS_idx = np.where(comp_dict['rem_id'] == 102)[0]
+            NS_kick_speed_scale=NS_kick_speed_mean/(2*np.sqrt(2/np.pi))
```

Comment:
Add some comment here, along the lines of (modify wording for clarity): 
The pdf of the Maxwellian is given by sqrt(2/pi) * x^2 * exp[-x^2/(2 * a^2)] / a^3, where "a" is defined as the scale parameter. The mean value of the Maxwellian is given by 2 * a * sqrt(2/pi). The scale parameter must then be modified to take the corresponding mean value.
Doesn't need to be too long, since we'll put a detailed explanation in documentation. But just something brief to clarify.

Reply:
Added:
```
            # Maxwellian pdf is sqrt(2/pi) * x^2 * exp[-x^2/(2 * a^2)] / a^3,
            # with a scale parameter `a`.
            # The Maxwellian mean is 2 * a * sqrt(2/pi).
            # Here we calculate the scipy.stats `scale` by dividing the
            # user defined mean by the Maxwellian mean.
```